ui/src/views/Tasks/ViewTask/index.jsx

```diff
@@ -735,19 +734,21 @@ export default class ViewTask extends Component {
               }
             />
             {tags.map(([key, value]) => (
-              <Chip
-                className={classes.tag}
-                key={key}
-                label={
-                  <Fragment>
-                    {key}
-                    :&nbsp;&nbsp;
+              <div key={key}>
+                <Chip
```

Comment:
Hm, I think this will insert a div for every chip:

```jsx
<div>
  <Chip />
</div>
<div>
  <Chip />
</div>
// ...
```

Sorry if I wasn't clear in https://github.com/taskcluster/taskcluster/pull/1513#discussion_r332202936. I'm looking more for something like

```jsx
<div>
  <Chip />
  <Chip />
  // ...
</div>
```

Reply:
Thanks @helfi92. I made that change, but I also tried rebasing to master and now it looks like my PR as 245 commits when I've only done a few, because it's grabbing other people's commits. Is that an issue? It might just be cleaner for me to close this PR and re-do this from the latest master.

Reply:
Sure, you can close and open a new PR. It's fine :)